README.md

```diff
@@ -70,9 +83,11 @@ fragment FullType on __Type {
 You can now pipe that file to the `query` command to send it.
 
 ```
-$ cat query.gql | circleci-beta query
+$ cat query.gql | circleci-cli query
 ```
 
+> You could also copy the query above, and run it directly from your Clipboard `pbpaste | circleci-cli query`.
```

Comment:
mac specific but probably okay (linux would be `xclip -sel clipboard -o | circleci-cli query`)